l3build.lua

```diff
@@ -28,31 +28,31 @@ for those people who are interested.
 release_date = "2020-06-04"
 
 -- File operations are aided by the LuaFileSystem module
+-- Next might be unnecessary on modern `texlua`
 local lfs = require("lfs")
 
 -- Local access to functions
 
-local assert           = assert
 local ipairs           = ipairs
 local insert           = table.insert
 local lookup           = kpse.lookup
 local match            = string.match
 local gsub             = string.gsub
-local next             = next
 local print            = print
-local select           = select
-local tonumber         = tonumber
 local exit             = os.exit
 
 -- l3build setup and functions
 kpse.set_program_name("kpsewhich")
-build_kpse_path = match(lookup("l3build.lua"),"(.*[/])")
```

Comment:
This again is breaking backward compatibility.

Reply:
Can I have more precision ?
I cannot find any mention of this in the repository.